detailed_docs/RC/resource_allocation.md

```diff
@@ -14,33 +14,40 @@ _Steps:_
 
 1. User chose to allow RC functionality via HMI and specified access mode as AUTO_ALLOW or didn't specify the access mode at all
 2. HMI sends notification to SDL that Remote control functionality is allowed with or without specified access mode
-3. RC_app_1 is in HMILevel FULL and is in control of the module_1 (sent request to either change settings or emulate button press)
-4. User activates RC_app_2 (app_2 is in FULL HMI level now)
-5. RC_app_2 sends control request to module_1 (sent request to either change settings or emulate button press)
+3. RC_app_1 is in HMILevel FULL sends request to allocate the module _1 (either  to change settings or emulate button press)
+4. User activates RC_app_2 (app_2 is in FULL HMI level now)	+4. RC_app_1 is in control of the the module_1
+5. RC_app_2 sends control request to module_1 (sent request to either change settings or emulate button press)	+5. SDL sends OnRCStatus to HMI and to registered application app1 and app2 with module_1 in section `allocatedModules` and remained modules in `freeModules`  
```

Comment:
```
SDL sends OnRCStatus to HMI and to registered application app1 and app2 with module_1 in section `allocatedModules` and remained modules in `freeModules` 
```
`module_1` shall be in only one `allocatedModules`  in notification to the app.
in this case, the notification to app1 include `module_1`  in `allocatedModules` , the notification to app2 does not include `module_1`  in `allocatedModules`

Reply:
@yang1070, thank you. 
please see updates https://github.com/smartdevicelink/sdl_requirements/pull/89/commits/bcefffcf4992deb88e3ab84717ac519cbd318197